### DISCUSSION TITLE
a question when i use this model in huggingface

### DISCUSSION THREAD

**Discussion by wjf364** [PixArt-alpha/PixArt-alpha#28] [Q&A] [answered]
when i try to inference this model in huggingface, i have encountered this problem:ValueError: Non-consecutive added token '<pad>' found. Should have index 32100 but has index 0 in saved vocabulary.

**Comment by wjf364** [comment]
how can i resolve it?

**Comment by yujincheng08** [comment, marked as answer]
upgrade transformer. there's no reason to stay in old version.
